src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/ChangeBasedEvent.java

```diff
@@ -75,49 +67,21 @@ public void setChange(Change change) {
         this.change = change;
     }
 
-/**
-     * The changed files in this patchset.
-     */
-    private List<String> files;
```

Comment:
Maybe I missed this one before, but this is also where old data will fail to serialize, so you need to do something similar to `Topic`. I.e. keep the old field but make it `transient` and add a `readResolve` that'll move the `files` into `Change`.

Reply:
will do